"2024-02-05 03:17:27,699 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-05 03:17:27,787 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-05 03:17:37,159 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-05 03:17:37,161 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-05 03:17:37,161 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 21:23:34,898 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 21:23:34,979 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:23:45,255 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849645 is handled. Duration 401 ms by bot id=6249292999"
"2024-02-19 21:23:47,544 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849646 is handled. Duration 862 ms by bot id=6249292999"
"2024-02-19 21:23:48,909 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849647 is handled. Duration 174 ms by bot id=6249292999"
"2024-02-19 21:23:49,899 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849648 is handled. Duration 136 ms by bot id=6249292999"
"2024-02-19 21:23:50,904 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849649 is handled. Duration 209 ms by bot id=6249292999"
"2024-02-19 21:23:51,480 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849650 is handled. Duration 188 ms by bot id=6249292999"
"2024-02-19 21:23:52,680 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849651 is handled. Duration 183 ms by bot id=6249292999"
"2024-02-19 21:23:53,745 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849652 is handled. Duration 188 ms by bot id=6249292999"
"2024-02-19 21:23:55,146 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 21:23:55,418 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849653 is handled. Duration 805 ms by bot id=6249292999"
"2024-02-19 21:25:08,788 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 21:25:08,788 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:25:08,788 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 21:25:12,592 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 21:25:12,673 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:25:14,823 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849654 is handled. Duration 695 ms by bot id=6249292999"
"2024-02-19 21:25:16,053 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849655 is handled. Duration 119 ms by bot id=6249292999"
"2024-02-19 21:25:17,207 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849656 is handled. Duration 123 ms by bot id=6249292999"
"2024-02-19 21:25:17,557 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849657 is handled. Duration 107 ms by bot id=6249292999"
"2024-02-19 21:25:17,904 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849658 is handled. Duration 115 ms by bot id=6249292999"
"2024-02-19 21:25:19,459 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849659 is handled. Duration 114 ms by bot id=6249292999"
"2024-02-19 21:25:20,861 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849660 is handled. Duration 133 ms by bot id=6249292999"
"2024-02-19 21:25:22,733 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 21:25:22,841 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849661 is handled. Duration 766 ms by bot id=6249292999"
"2024-02-19 21:25:31,981 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 21:25:31,981 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:25:31,982 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 21:25:35,718 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 21:25:35,805 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:25:37,975 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849662 is handled. Duration 1286 ms by bot id=6249292999"
"2024-02-19 21:25:38,305 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849664 is handled. Duration 216 ms by bot id=6249292999"
"2024-02-19 21:25:38,351 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849663 is handled. Duration 378 ms by bot id=6249292999"
"2024-02-19 21:25:38,538 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849665 is handled. Duration 216 ms by bot id=6249292999"
"2024-02-19 21:25:38,896 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849666 is handled. Duration 259 ms by bot id=6249292999"
"2024-02-19 21:25:39,107 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849667 is handled. Duration 309 ms by bot id=6249292999"
"2024-02-19 21:25:41,447 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849668 is handled. Duration 181 ms by bot id=6249292999"
"2024-02-19 21:25:44,299 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 21:25:44,513 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849669 is handled. Duration 728 ms by bot id=6249292999"
"2024-02-19 21:25:56,483 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849670 is not handled. Duration 0 ms by bot id=6249292999"
"2024-02-19 21:26:08,831 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 21:26:08,831 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:26:08,832 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 21:55:21,198 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 21:55:21,283 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:55:30,795 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849671 is handled. Duration 774 ms by bot id=6249292999"
"2024-02-19 21:55:40,702 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 21:55:40,703 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 21:55:40,703 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:01:29,101 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:01:29,179 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:01:41,956 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849672 is handled. Duration 265 ms by bot id=6249292999"
"2024-02-19 22:01:43,878 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849673 is handled. Duration 1080 ms by bot id=6249292999"
"2024-02-19 22:01:44,555 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849674 is handled. Duration 192 ms by bot id=6249292999"
"2024-02-19 22:01:45,442 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849675 is handled. Duration 262 ms by bot id=6249292999"
"2024-02-19 22:01:46,155 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849676 is handled. Duration 271 ms by bot id=6249292999"
"2024-02-19 22:01:46,865 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849677 is handled. Duration 205 ms by bot id=6249292999"
"2024-02-19 22:01:47,721 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849678 is handled. Duration 213 ms by bot id=6249292999"
"2024-02-19 22:01:48,802 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849679 is handled. Duration 220 ms by bot id=6249292999"
"2024-02-19 22:01:50,215 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:01:50,399 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849680 is handled. Duration 673 ms by bot id=6249292999"
"2024-02-19 22:01:52,804 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849681 is not handled. Duration 512 ms by bot id=6249292999"
"2024-02-19 22:01:52,805 - [ERROR] - aiogram.event - (dispatcher.py)._process_update(315) - Cause exception while process update id=177849681 by bot id=6249292999\nTelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '-' is reserved and must be escaped with the preceding '\\'\nTraceback (most recent call last):\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 309, in _process_update\n    response = await self.feed_update(bot, update, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 158, in feed_update\n    response = await self.update.wrap_outer_middleware(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py\", line 25, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py\", line 27, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/fsm/middleware.py\", line 41, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 276, in _listen_update\n    return await self.propagate_event(update_type=update_type, event=event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 128, in propagate_event\n    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 123, in _wrapped\n    return await self._propagate_event(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 148, in _propagate_event\n    response = await observer.trigger(event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Desktop/Projects/work/Victoria_bot_v2/utils/callbacks/bot_callbacks.py\", line 44, in new_link\n    await query.message.answer('\u041f\u043e\u043d\u044f\u043b, \u0432\u0432\u0435\u0434\u0438 \u0430\u0431\u0441\u043e\u043b\u044e\u0442\u043d\u043e \u043b\u044e\u0431\u043e\u0439 \u0432\u0435\u0431\u0445\u0443\u043a \u0432 \u0442\u0430\u043a\u043e\u043c \u0444\u043e\u0440\u043c\u0430\u0442\u0435 - https://b24-test.bitrix24.ru/rest/1/2k2rrk0gfqyb4by2/,'\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/methods/base.py\", line 84, in emit\n    return await bot(self)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/bot.py\", line 441, in __call__\n    return await self.session(self, method, timeout=request_timeout)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 257, in __call__\n    return cast(TelegramType, await middleware(bot, method))\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/aiohttp.py\", line 179, in make_request\n    response = self.check_response(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 120, in check_response\n    raise TelegramBadRequest(method=method, message=description)\naiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '-' is reserved and must be escaped with the preceding '\\'"
"2024-02-19 22:02:35,700 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:02:35,701 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:02:35,701 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:02:39,598 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:02:39,675 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:02:42,821 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849682 is handled. Duration 349 ms by bot id=6249292999"
"2024-02-19 22:02:44,666 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849683 is handled. Duration 961 ms by bot id=6249292999"
"2024-02-19 22:02:45,959 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849684 is handled. Duration 384 ms by bot id=6249292999"
"2024-02-19 22:02:46,362 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849685 is handled. Duration 325 ms by bot id=6249292999"
"2024-02-19 22:02:46,771 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849686 is handled. Duration 372 ms by bot id=6249292999"
"2024-02-19 22:02:47,137 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849687 is handled. Duration 312 ms by bot id=6249292999"
"2024-02-19 22:02:48,272 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849688 is handled. Duration 119 ms by bot id=6249292999"
"2024-02-19 22:02:49,285 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849689 is handled. Duration 117 ms by bot id=6249292999"
"2024-02-19 22:02:51,510 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:02:51,691 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849690 is handled. Duration 595 ms by bot id=6249292999"
"2024-02-19 22:02:54,196 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849691 is not handled. Duration 634 ms by bot id=6249292999"
"2024-02-19 22:02:54,197 - [ERROR] - aiogram.event - (dispatcher.py)._process_update(315) - Cause exception while process update id=177849691 by bot id=6249292999\nTelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '-' is reserved and must be escaped with the preceding '\\'\nTraceback (most recent call last):\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 309, in _process_update\n    response = await self.feed_update(bot, update, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 158, in feed_update\n    response = await self.update.wrap_outer_middleware(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py\", line 25, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py\", line 27, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/fsm/middleware.py\", line 41, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 276, in _listen_update\n    return await self.propagate_event(update_type=update_type, event=event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 128, in propagate_event\n    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 123, in _wrapped\n    return await self._propagate_event(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 148, in _propagate_event\n    response = await observer.trigger(event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Desktop/Projects/work/Victoria_bot_v2/utils/callbacks/bot_callbacks.py\", line 44, in new_link\n    await query.message.answer('\u041f\u043e\u043d\u044f\u043b, \u0432\u0432\u0435\u0434\u0438 \u0430\u0431\u0441\u043e\u043b\u044e\u0442\u043d\u043e \u043b\u044e\u0431\u043e\u0439 \u0432\u0435\u0431\u0445\u0443\u043a \u0432 \u0442\u0430\u043a\u043e\u043c \u0444\u043e\u0440\u043c\u0430\u0442\u0435 \\- https://b24-test.bitrix24.ru/rest/1/2k2rrk0gfqyb4by2/,'\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/methods/base.py\", line 84, in emit\n    return await bot(self)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/bot.py\", line 441, in __call__\n    return await self.session(self, method, timeout=request_timeout)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 257, in __call__\n    return cast(TelegramType, await middleware(bot, method))\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/aiohttp.py\", line 179, in make_request\n    response = self.check_response(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 120, in check_response\n    raise TelegramBadRequest(method=method, message=description)\naiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '-' is reserved and must be escaped with the preceding '\\'"
"2024-02-19 22:03:22,587 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:03:22,588 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:03:22,588 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:03:26,754 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:03:26,833 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:03:27,695 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849692 is handled. Duration 359 ms by bot id=6249292999"
"2024-02-19 22:03:29,582 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849693 is handled. Duration 816 ms by bot id=6249292999"
"2024-02-19 22:03:30,503 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849694 is handled. Duration 144 ms by bot id=6249292999"
"2024-02-19 22:03:30,926 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849695 is handled. Duration 198 ms by bot id=6249292999"
"2024-02-19 22:03:31,191 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849696 is handled. Duration 180 ms by bot id=6249292999"
"2024-02-19 22:03:31,491 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849697 is handled. Duration 178 ms by bot id=6249292999"
"2024-02-19 22:03:31,882 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849698 is handled. Duration 213 ms by bot id=6249292999"
"2024-02-19 22:03:33,258 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849699 is handled. Duration 268 ms by bot id=6249292999"
"2024-02-19 22:03:34,733 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:03:35,050 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849700 is handled. Duration 814 ms by bot id=6249292999"
"2024-02-19 22:03:36,726 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849701 is not handled. Duration 799 ms by bot id=6249292999"
"2024-02-19 22:03:36,726 - [ERROR] - aiogram.event - (dispatcher.py)._process_update(315) - Cause exception while process update id=177849701 by bot id=6249292999\nTelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '.' is reserved and must be escaped with the preceding '\\'\nTraceback (most recent call last):\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 309, in _process_update\n    response = await self.feed_update(bot, update, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 158, in feed_update\n    response = await self.update.wrap_outer_middleware(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py\", line 25, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py\", line 27, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/fsm/middleware.py\", line 41, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 276, in _listen_update\n    return await self.propagate_event(update_type=update_type, event=event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 128, in propagate_event\n    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 123, in _wrapped\n    return await self._propagate_event(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 148, in _propagate_event\n    response = await observer.trigger(event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Desktop/Projects/work/Victoria_bot_v2/utils/callbacks/bot_callbacks.py\", line 44, in new_link\n    await query.message.answer('\u041f\u043e\u043d\u044f\u043b, \u0432\u0432\u0435\u0434\u0438 \u0430\u0431\u0441\u043e\u043b\u044e\u0442\u043d\u043e \u043b\u044e\u0431\u043e\u0439 \u0432\u0435\u0431\u0445\u0443\u043a \u0432 \u0442\u0430\u043a\u043e\u043c \u0444\u043e\u0440\u043c\u0430\u0442\u0435 \\- https://b24\\-test.bitrix24\\.ru/rest/1/2k2rrk0gfqyb4by2/,'\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/methods/base.py\", line 84, in emit\n    return await bot(self)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/bot.py\", line 441, in __call__\n    return await self.session(self, method, timeout=request_timeout)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 257, in __call__\n    return cast(TelegramType, await middleware(bot, method))\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/aiohttp.py\", line 179, in make_request\n    response = self.check_response(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 120, in check_response\n    raise TelegramBadRequest(method=method, message=description)\naiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '.' is reserved and must be escaped with the preceding '\\'"
"2024-02-19 22:03:59,802 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:03:59,802 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:03:59,802 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:04:03,557 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:04:03,636 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:04:05,523 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849702 is handled. Duration 309 ms by bot id=6249292999"
"2024-02-19 22:04:08,305 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849703 is handled. Duration 300 ms by bot id=6249292999"
"2024-02-19 22:04:11,430 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849704 is handled. Duration 796 ms by bot id=6249292999"
"2024-02-19 22:04:13,637 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849705 is handled. Duration 198 ms by bot id=6249292999"
"2024-02-19 22:04:14,794 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849706 is handled. Duration 161 ms by bot id=6249292999"
"2024-02-19 22:04:15,180 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849707 is handled. Duration 169 ms by bot id=6249292999"
"2024-02-19 22:04:15,551 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849708 is handled. Duration 189 ms by bot id=6249292999"
"2024-02-19 22:04:15,881 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849709 is handled. Duration 222 ms by bot id=6249292999"
"2024-02-19 22:04:16,343 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849710 is handled. Duration 224 ms by bot id=6249292999"
"2024-02-19 22:04:16,850 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:04:17,116 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849711 is handled. Duration 751 ms by bot id=6249292999"
"2024-02-19 22:04:19,897 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849712 is not handled. Duration 718 ms by bot id=6249292999"
"2024-02-19 22:04:19,897 - [ERROR] - aiogram.event - (dispatcher.py)._process_update(315) - Cause exception while process update id=177849712 by bot id=6249292999\nTelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '.' is reserved and must be escaped with the preceding '\\'\nTraceback (most recent call last):\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 309, in _process_update\n    response = await self.feed_update(bot, update, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 158, in feed_update\n    response = await self.update.wrap_outer_middleware(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py\", line 25, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py\", line 27, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/fsm/middleware.py\", line 41, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 276, in _listen_update\n    return await self.propagate_event(update_type=update_type, event=event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 128, in propagate_event\n    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 123, in _wrapped\n    return await self._propagate_event(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 148, in _propagate_event\n    response = await observer.trigger(event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Desktop/Projects/work/Victoria_bot_v2/utils/callbacks/bot_callbacks.py\", line 44, in new_link\n    await query.message.answer('\u041f\u043e\u043d\u044f\u043b, \u0432\u0432\u0435\u0434\u0438 \u0430\u0431\u0441\u043e\u043b\u044e\u0442\u043d\u043e \u043b\u044e\u0431\u043e\u0439 \u0432\u0435\u0431\u0445\u0443\u043a \u0432 \u0442\u0430\u043a\u043e\u043c \u0444\u043e\u0440\u043c\u0430\u0442\u0435 \\- https\\://b24\\-test.bitrix24\\.ru/rest/1/2k2rrk0gfqyb4by2/,'\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/methods/base.py\", line 84, in emit\n    return await bot(self)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/bot.py\", line 441, in __call__\n    return await self.session(self, method, timeout=request_timeout)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 257, in __call__\n    return cast(TelegramType, await middleware(bot, method))\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/aiohttp.py\", line 179, in make_request\n    response = self.check_response(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 120, in check_response\n    raise TelegramBadRequest(method=method, message=description)\naiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '.' is reserved and must be escaped with the preceding '\\'"
"2024-02-19 22:04:47,112 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:04:47,113 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:04:47,113 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:04:51,131 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:04:51,210 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:04:53,589 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849714 is handled. Duration 272 ms by bot id=6249292999"
"2024-02-19 22:04:55,020 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849715 is handled. Duration 679 ms by bot id=6249292999"
"2024-02-19 22:04:56,691 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849716 is handled. Duration 200 ms by bot id=6249292999"
"2024-02-19 22:04:56,914 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849717 is handled. Duration 188 ms by bot id=6249292999"
"2024-02-19 22:04:57,130 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849718 is handled. Duration 151 ms by bot id=6249292999"
"2024-02-19 22:04:57,498 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849719 is handled. Duration 170 ms by bot id=6249292999"
"2024-02-19 22:04:57,757 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849720 is handled. Duration 137 ms by bot id=6249292999"
"2024-02-19 22:04:58,082 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849721 is handled. Duration 179 ms by bot id=6249292999"
"2024-02-19 22:05:02,083 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:05:02,324 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849722 is handled. Duration 672 ms by bot id=6249292999"
"2024-02-19 22:05:04,125 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849723 is handled. Duration 719 ms by bot id=6249292999"
"2024-02-19 22:05:32,208 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:05:32,209 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:05:32,209 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:05:36,041 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:05:36,122 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:05:37,005 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849724 is handled. Duration 270 ms by bot id=6249292999"
"2024-02-19 22:05:38,572 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849725 is handled. Duration 744 ms by bot id=6249292999"
"2024-02-19 22:05:39,603 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849726 is handled. Duration 132 ms by bot id=6249292999"
"2024-02-19 22:05:40,065 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849727 is handled. Duration 162 ms by bot id=6249292999"
"2024-02-19 22:05:40,544 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849728 is handled. Duration 179 ms by bot id=6249292999"
"2024-02-19 22:05:41,016 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849729 is handled. Duration 155 ms by bot id=6249292999"
"2024-02-19 22:05:42,029 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849730 is handled. Duration 242 ms by bot id=6249292999"
"2024-02-19 22:05:43,782 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849731 is handled. Duration 115 ms by bot id=6249292999"
"2024-02-19 22:05:45,191 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:05:45,346 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849732 is handled. Duration 551 ms by bot id=6249292999"
"2024-02-19 22:05:47,628 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849733 is handled. Duration 479 ms by bot id=6249292999"
"2024-02-19 22:06:05,619 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849734 is not handled. Duration 2 ms by bot id=6249292999"
"2024-02-19 22:06:05,620 - [ERROR] - aiogram.event - (dispatcher.py)._process_update(315) - Cause exception while process update id=177849734 by bot id=6249292999\nAttributeError: 'NoneType' object has no attribute 'new_link'\nTraceback (most recent call last):\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 309, in _process_update\n    response = await self.feed_update(bot, update, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 158, in feed_update\n    response = await self.update.wrap_outer_middleware(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py\", line 25, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py\", line 27, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/fsm/middleware.py\", line 41, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 276, in _listen_update\n    return await self.propagate_event(update_type=update_type, event=event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 128, in propagate_event\n    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 123, in _wrapped\n    return await self._propagate_event(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 148, in _propagate_event\n    response = await observer.trigger(event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Desktop/Projects/work/Victoria_bot_v2/bot_messages/bitrix_messages.py\", line 138, in new_bitrix_link\n    info.new_link = get_data.get('new_link')\nAttributeError: 'NoneType' object has no attribute 'new_link'"
"2024-02-19 22:08:38,277 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:08:38,277 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:08:38,278 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:08:42,145 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:08:42,227 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:08:42,974 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849735 is handled. Duration 255 ms by bot id=6249292999"
"2024-02-19 22:08:44,667 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849736 is handled. Duration 593 ms by bot id=6249292999"
"2024-02-19 22:08:45,520 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849737 is handled. Duration 133 ms by bot id=6249292999"
"2024-02-19 22:08:45,990 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849738 is handled. Duration 102 ms by bot id=6249292999"
"2024-02-19 22:08:46,606 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849739 is handled. Duration 107 ms by bot id=6249292999"
"2024-02-19 22:08:47,214 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849740 is handled. Duration 125 ms by bot id=6249292999"
"2024-02-19 22:08:47,815 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849741 is handled. Duration 104 ms by bot id=6249292999"
"2024-02-19 22:08:48,632 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849742 is handled. Duration 115 ms by bot id=6249292999"
"2024-02-19 22:08:50,087 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:08:50,262 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849743 is handled. Duration 539 ms by bot id=6249292999"
"2024-02-19 22:08:52,269 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849744 is handled. Duration 589 ms by bot id=6249292999"
"2024-02-19 22:08:54,193 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849745 is not handled. Duration 3 ms by bot id=6249292999"
"2024-02-19 22:08:54,193 - [ERROR] - aiogram.event - (dispatcher.py)._process_update(315) - Cause exception while process update id=177849745 by bot id=6249292999\nAttributeError: 'NoneType' object has no attribute 'new_link'\nTraceback (most recent call last):\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 309, in _process_update\n    response = await self.feed_update(bot, update, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 158, in feed_update\n    response = await self.update.wrap_outer_middleware(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py\", line 25, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py\", line 27, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/fsm/middleware.py\", line 41, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 276, in _listen_update\n    return await self.propagate_event(update_type=update_type, event=event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 128, in propagate_event\n    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 123, in _wrapped\n    return await self._propagate_event(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 148, in _propagate_event\n    response = await observer.trigger(event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Desktop/Projects/work/Victoria_bot_v2/bot_messages/bitrix_messages.py\", line 138, in new_bitrix_link\n    info.new_link = get_data.get('new_link')\nAttributeError: 'NoneType' object has no attribute 'new_link'"
"2024-02-19 22:09:15,936 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:09:15,936 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:09:15,936 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:24:20,017 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:24:20,104 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:24:28,320 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849747 is handled. Duration 325 ms by bot id=6249292999"
"2024-02-19 22:24:30,215 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849748 is handled. Duration 845 ms by bot id=6249292999"
"2024-02-19 22:24:44,821 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849749 is handled. Duration 98 ms by bot id=6249292999"
"2024-02-19 22:24:45,569 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849750 is handled. Duration 104 ms by bot id=6249292999"
"2024-02-19 22:24:46,371 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849751 is handled. Duration 125 ms by bot id=6249292999"
"2024-02-19 22:24:47,124 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849752 is handled. Duration 138 ms by bot id=6249292999"
"2024-02-19 22:24:47,618 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849753 is handled. Duration 141 ms by bot id=6249292999"
"2024-02-19 22:24:47,994 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849754 is handled. Duration 116 ms by bot id=6249292999"
"2024-02-19 22:24:48,664 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:24:48,788 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849755 is handled. Duration 565 ms by bot id=6249292999"
"2024-02-19 22:24:50,794 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849756 is handled. Duration 385 ms by bot id=6249292999"
"2024-02-19 22:24:52,143 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849757 is handled. Duration 169 ms by bot id=6249292999"
"2024-02-19 22:33:23,741 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849758 is handled. Duration 814 ms by bot id=6249292999"
"2024-02-19 22:33:25,705 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849759 is handled. Duration 268 ms by bot id=6249292999"
"2024-02-19 22:33:27,192 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849760 is handled. Duration 304 ms by bot id=6249292999"
"2024-02-19 22:33:28,237 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849761 is handled. Duration 224 ms by bot id=6249292999"
"2024-02-19 22:33:28,848 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849762 is handled. Duration 131 ms by bot id=6249292999"
"2024-02-19 22:33:29,566 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849763 is handled. Duration 118 ms by bot id=6249292999"
"2024-02-19 22:33:30,257 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849764 is handled. Duration 116 ms by bot id=6249292999"
"2024-02-19 22:33:31,440 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:33:31,555 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849765 is handled. Duration 456 ms by bot id=6249292999"
"2024-02-19 22:33:33,633 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849766 is handled. Duration 385 ms by bot id=6249292999"
"2024-02-19 22:33:37,863 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849767 is handled. Duration 168 ms by bot id=6249292999"
"2024-02-19 22:33:47,076 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849768 is handled. Duration 544 ms by bot id=6249292999"
"2024-02-19 22:33:48,944 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849769 is handled. Duration 105 ms by bot id=6249292999"
"2024-02-19 22:33:50,317 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849770 is handled. Duration 111 ms by bot id=6249292999"
"2024-02-19 22:33:51,136 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849771 is handled. Duration 121 ms by bot id=6249292999"
"2024-02-19 22:33:52,044 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849772 is handled. Duration 123 ms by bot id=6249292999"
"2024-02-19 22:33:53,308 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849773 is handled. Duration 234 ms by bot id=6249292999"
"2024-02-19 22:33:54,366 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849774 is handled. Duration 106 ms by bot id=6249292999"
"2024-02-19 22:33:56,116 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:33:56,280 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849775 is handled. Duration 472 ms by bot id=6249292999"
"2024-02-19 22:33:58,121 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849776 is handled. Duration 936 ms by bot id=6249292999"
"2024-02-19 22:33:59,875 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849777 is handled. Duration 103 ms by bot id=6249292999"
"2024-02-19 22:35:46,466 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:35:46,467 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:35:46,467 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:35:50,794 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:35:50,872 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:35:54,014 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849778 is handled. Duration 788 ms by bot id=6249292999"
"2024-02-19 22:35:55,925 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849779 is handled. Duration 158 ms by bot id=6249292999"
"2024-02-19 22:35:57,080 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849780 is handled. Duration 195 ms by bot id=6249292999"
"2024-02-19 22:35:58,013 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849781 is handled. Duration 152 ms by bot id=6249292999"
"2024-02-19 22:35:59,012 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849782 is handled. Duration 107 ms by bot id=6249292999"
"2024-02-19 22:36:00,160 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849783 is handled. Duration 203 ms by bot id=6249292999"
"2024-02-19 22:36:02,290 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849784 is handled. Duration 195 ms by bot id=6249292999"
"2024-02-19 22:36:04,623 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'INVALID_CREDENTIALS', 'error_description': 'Invalid request credentials'}"
"2024-02-19 22:36:04,774 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849785 is handled. Duration 608 ms by bot id=6249292999"
"2024-02-19 22:36:07,387 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849786 is not handled. Duration 671 ms by bot id=6249292999"
"2024-02-19 22:36:07,388 - [ERROR] - aiogram.event - (dispatcher.py)._process_update(315) - Cause exception while process update id=177849786 by bot id=6249292999\nTelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '-' is reserved and must be escaped with the preceding '\\'\nTraceback (most recent call last):\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 309, in _process_update\n    response = await self.feed_update(bot, update, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 158, in feed_update\n    response = await self.update.wrap_outer_middleware(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py\", line 25, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py\", line 27, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/fsm/middleware.py\", line 41, in __call__\n    return await handler(event, data)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py\", line 276, in _listen_update\n    return await self.propagate_event(update_type=update_type, event=event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 128, in propagate_event\n    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 123, in _wrapped\n    return await self._propagate_event(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/router.py\", line 148, in _propagate_event\n    response = await observer.trigger(event, **kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py\", line 121, in trigger\n    return await wrapped_inner(event, kwargs)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py\", line 43, in call\n    return await wrapped()\n  File \"/home/denis/Desktop/Projects/work/Victoria_bot_v2/utils/callbacks/bot_callbacks.py\", line 44, in new_link\n    await query.message.answer('\u041f\u043e\u043d\u044f\u043b, \u0432\u0432\u0435\u0434\u0438 \u043d\u043e\u0432\u0443\u044e \u0441\u0441\u044b\u043b\u043a\u0443, \u0432\u043e\u0437\u044c\u043c\u0438 \u0435\u0451 \u0438\u0437 \u043f\u043e\u043b\u044f - \u0412\u0435\u0431\u0445\u0443\u043a \u0434\u043b\u044f \u0432\u044b\u0437\u043e\u0432\u0430 rest api')\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/methods/base.py\", line 84, in emit\n    return await bot(self)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/bot.py\", line 441, in __call__\n    return await self.session(self, method, timeout=request_timeout)\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 257, in __call__\n    return cast(TelegramType, await middleware(bot, method))\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/aiohttp.py\", line 179, in make_request\n    response = self.check_response(\n  File \"/home/denis/Env/telegram_bot_viktoria/lib/python3.10/site-packages/aiogram/client/session/base.py\", line 120, in check_response\n    raise TelegramBadRequest(method=method, message=description)\naiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Character '-' is reserved and must be escaped with the preceding '\\'"
"2024-02-19 22:36:15,156 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:36:15,157 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:36:15,157 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:36:18,829 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:36:18,909 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:36:19,983 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849787 is handled. Duration 295 ms by bot id=6249292999"
"2024-02-19 22:36:21,773 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849788 is handled. Duration 648 ms by bot id=6249292999"
"2024-02-19 22:36:23,131 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849789 is handled. Duration 184 ms by bot id=6249292999"
"2024-02-19 22:36:24,193 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849790 is handled. Duration 147 ms by bot id=6249292999"
"2024-02-19 22:36:25,311 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849791 is handled. Duration 158 ms by bot id=6249292999"
"2024-02-19 22:36:26,178 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849792 is handled. Duration 133 ms by bot id=6249292999"
"2024-02-19 22:36:27,239 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849793 is handled. Duration 169 ms by bot id=6249292999"
"2024-02-19 22:36:28,693 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849794 is handled. Duration 160 ms by bot id=6249292999"
"2024-02-19 22:36:29,947 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'INVALID_CREDENTIALS', 'error_description': 'Invalid request credentials'}"
"2024-02-19 22:36:30,142 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849795 is handled. Duration 528 ms by bot id=6249292999"
"2024-02-19 22:36:31,590 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849796 is handled. Duration 608 ms by bot id=6249292999"
"2024-02-19 22:36:40,215 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849797 is handled. Duration 213 ms by bot id=6249292999"
"2024-02-19 22:37:11,028 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849798 is handled. Duration 850 ms by bot id=6249292999"
"2024-02-19 22:37:12,079 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849799 is handled. Duration 127 ms by bot id=6249292999"
"2024-02-19 22:37:13,037 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849800 is handled. Duration 220 ms by bot id=6249292999"
"2024-02-19 22:37:13,869 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849801 is handled. Duration 189 ms by bot id=6249292999"
"2024-02-19 22:37:15,043 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849802 is handled. Duration 181 ms by bot id=6249292999"
"2024-02-19 22:37:16,247 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849803 is handled. Duration 195 ms by bot id=6249292999"
"2024-02-19 22:37:17,322 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849804 is handled. Duration 237 ms by bot id=6249292999"
"2024-02-19 22:37:18,886 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'INVALID_CREDENTIALS', 'error_description': 'Invalid request credentials'}"
"2024-02-19 22:37:19,015 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849805 is handled. Duration 587 ms by bot id=6249292999"
"2024-02-19 22:40:13,594 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:40:13,595 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:40:13,595 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
"2024-02-19 22:40:50,881 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(527) - Start polling"
"2024-02-19 22:40:50,977 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(341) - Run polling for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:43:39,932 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849807 is handled. Duration 311 ms by bot id=6249292999"
"2024-02-19 22:43:40,116 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849808 is handled. Duration 115 ms by bot id=6249292999"
"2024-02-19 22:43:40,641 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849809 is handled. Duration 104 ms by bot id=6249292999"
"2024-02-19 22:43:41,152 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849810 is handled. Duration 128 ms by bot id=6249292999"
"2024-02-19 22:43:43,332 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849811 is handled. Duration 634 ms by bot id=6249292999"
"2024-02-19 22:43:44,619 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849812 is handled. Duration 172 ms by bot id=6249292999"
"2024-02-19 22:43:45,278 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849813 is handled. Duration 229 ms by bot id=6249292999"
"2024-02-19 22:43:45,789 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849814 is handled. Duration 196 ms by bot id=6249292999"
"2024-02-19 22:43:46,254 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849815 is handled. Duration 169 ms by bot id=6249292999"
"2024-02-19 22:43:46,866 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849816 is handled. Duration 171 ms by bot id=6249292999"
"2024-02-19 22:43:47,377 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849817 is handled. Duration 192 ms by bot id=6249292999"
"2024-02-19 22:43:48,111 - [INFO] - STDOUT - (bot_logger.py).write(20) - \u041e\u0448\u0438\u0431\u043a\u0430: {'error': 'ACCESS_DENIED', 'error_description': 'REST is available only on commercial plans.'}"
"2024-02-19 22:43:48,310 - [INFO] - aiogram.event - (dispatcher.py).feed_update(172) - Update id=177849818 is handled. Duration 603 ms by bot id=6249292999"
"2024-02-19 22:45:57,982 - [WARNING] - aiogram.dispatcher - (dispatcher.py)._signal_stop_polling(455) - Received SIGINT signal"
"2024-02-19 22:45:57,982 - [INFO] - aiogram.dispatcher - (dispatcher.py)._polling(359) - Polling stopped for bot @denis14101999_first_bot id=6249292999 - 'Billy_bot'"
"2024-02-19 22:45:57,982 - [INFO] - aiogram.dispatcher - (dispatcher.py).start_polling(554) - Polling stopped"
